doc/users/dflt_style_changes.rst

```diff
@@ -1065,6 +1065,17 @@ Z-order
 
 
 
+``ScalarFormatter`` tick label formatting with offsets
+======================================================
+
+With the default of ``rcParams['axes.formatter.useoffset'] = True``,
+an offset will be used when it will save 4 or more digits.  This can
```

Comment:
I introduced that terminology (see last chunk of https://github.com/matplotlib/matplotlib/pull/5785/files).  It's not "retain", it's the leading digits that are *not* retained thanks to the use of the offset (i.e., exactly what @efiring said).